app/src/main/java/com/nearsoft/referralsapp/JobListing.java

```diff
@@ -0,0 +1,82 @@
+package com.nearsoft.referralsapp;
+
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.util.Log;
+import android.widget.TextView;
+
+import com.android.volley.Request;
+import com.android.volley.RequestQueue;
+import com.android.volley.Response;
+import com.android.volley.VolleyError;
+import com.android.volley.toolbox.StringRequest;
+import com.android.volley.toolbox.Volley;
+
+import java.util.ArrayList;
+
+public class JobListing extends AppCompatActivity {
+
+    private RequestQueue mRequestQueue;
+    private static final String TAG = "TAG";
+    private RecyclerView mRecyclerView;
+    private LinearLayoutManager mLayoutManager;
+    private RecyclerView.Adapter mAdapter;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_job_listing);
+        mRecyclerView = (RecyclerView) findViewById(R.id.recycler_view);
+
+        // Instantiate the RequestQueue.
+        mRequestQueue = Volley.newRequestQueue(this);
+        String url ="https://my-json-server.typicode.com/haydeeR/API-test/positions";
+
+        // Request a string response from the provided URL.
+        StringRequest stringRequest = new StringRequest(Request.Method.GET, url,
+                new Response.Listener<String>() {
+                    @Override
+                    public void onResponse(String response) {
+                        ArrayList<String> result = new ArrayList<String>();
```

Comment:
Create a class to map the response and use a class like Gson to parse the content

Reply:
It was not necessary to map the response because I am only getting a single attribute from the Json Object.